ToDoManager.java

```diff
@@ -77,6 +78,22 @@ public static void executeUserCommand(String userCommand) {
 					System.out.println("Can not create new To Do. User must provide a description");
 				}
 				break;
+			case "complete":
```

Comment:
Cleaned up the try/catch block.  Also changed the commands to Enums.  Just did a quick google so I'm not sure if this is how its supposed to be used, but the program doesn't bork, which is nice.